pages/about.html

```diff
@@ -0,0 +1,159 @@
+        </p>
+        <p class="font-2 color-2 mail-header">
+          Please contact us via email for further inquiries
+        </p>
+        <a class="mail" href="mailto:help@culturecustodian.com">questions@globalenegysummit.com</a>
+      </div>
+    </header>
+    <section class="summit-details">
+      <h2 class="font-2 color-2 sub-header">The Global Energy Summit Logo</h2>
+      <div class="red-line"></div>
+      <p class="font-2 color-2">
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+      </p>
+      <p class="font-2 color-2 desktop-only">
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+        dummy text ever since the 1500s.
+      </p>
+      <span class="cc-logo">
+        <img src="../images/back3.jpg" alt="Logo">
+      </span>
+    </section>
+    <section id="events" class="past-events">
+      <h3 class="font-2 color-2 sub-header">
+        Past Energy Global Events
+      </h3>
+      <div class="red-line"></div>
+      <p class="color-2 font-2">
+        Take a look at the last two energy summits that took place in England and France
+      </p>
+      <div class="previous-events">
+        <div class="e-2019 event">
+          <h4 class=" font-2">2013</h4>
+          <p class=" font-2">Culture Custodian Global Energy Summit in London, Uk</p>
+        </div>
+        <div class="e-2021 event">
+          <h4 class="font-2">2020</h4>
+          <p class="font-2">Culture Custodian Global Summit in Paris, France</p>
+        </div>
+      </div>
+    </section>
+    <section class="partners mobile-only">
+      <h6 class="font-2 color-3">Partners</h6>
+      <div class="red-line"></div>
+      <div class="partner-logo">
+        <img class="" src="../images/partners/nike-logo.svg" alt="nike logo">
+        <img class="" src="../images/partners/addidas.png" alt="adiddas logo">
+        <img class="" src="../images/partners/google-logo.svg" alt="google logo">
+        <img class="" src="../images/partners/mozilla.svg" alt="mozilla logo">
+        <img class="" src="../images/partners/gucci.svg" alt="gucci logo">
+      </div>
+    </section>
+    <footer>
+      <span class="footer-logo mobile-only">
+        <img src="../images/lg5.png" alt="Logo">
+      </span>
+      <span class="footer-logo desktop-only">
+        <img src="../images/lg5.png" alt="Logo">
+      </span>
+      <div class="footer-text">
+        <p class="font-2 color-2">
+          2022 Global Energy summit. Some Rights Reserved.
+        </p>
+        <p class="desktop-only font-2 color-2">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora atque numquam dolores?
+          <br>lorem
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Expedita.
+        </p>
+      </div>
+    </footer>
```

Comment:
- [ ] Please, consider fixing of footer here on the About page desktop version also. It is also, as on Main page, has an alignment issue and the logo distorted 👍
<img width="1430" alt="Screen Shot 2022-03-15 at 12 09 31" src="https://user-images.githubusercontent.com/49064106/158333882-4ac174f9-e775-45c1-b5dc-6aee1c53f3e1.png">